src/pages/add-card/payment-input/components/cardPreview/CardPreview.tsx

```diff
@@ -32,13 +34,20 @@ function CardPreview({ cardNumbers, expirationDate }: CardInformationType) {
   }, [cardNumbers]);
 
   return (
-    <div className={styles.container}>
+    <div
+      style={
+        brandName === ""
+          ? {}
+          : { backgroundColor: CARD_BACKGROUND_COLOR[brandName] }
+      }
```

Comment:
```suggestion
     style={{
          backgroundColor: brandName ? CARD_BACKGROUND_COLOR[brandName] : undefined,
      }}
```
큰차이는 아니지만 요게 좀 더 깔끔할 것 같은데 어떠신가요?

Reply:
오오 좋습니다! 이게 더 가독성도 좋고 깔끔한 것 같아요!

[f5d5534](https://github.com/woowacourse/react-payments/pull/477/commits/f5d55345ea93f534daaaddfb03e0f5920b227dab)

반영하였습니다 😄👍